packages/data-stores/src/support-queries/use-submit-support-ticket.ts

```diff
@@ -7,6 +7,7 @@ type Ticket = {
 	locale: string;
 	client: string;
 	is_chat_overflow: boolean;
+	is_help_center: boolean;
```

Comment:
What do you think about adding a more general `source` value instead? If `source === 'help-center'` then we add a tag on the backend, etc. But once we have that source, we can use it for other useful things, like tracking how many requests we have from each source, adding it to server-side track events, etc.

Reply:
Great idea!